stock_picking_group_by_partner_by_carrier/models/sale_order.py

```diff
@@ -13,6 +13,10 @@ def action_cancel(self):
                 SaleOrder, sale_order.with_context(cancel_sale_id=sale_order.id)
             ).action_cancel()
 
+    def action_draft(self):
+        super().action_draft()
```

Comment:
As Odoo standard function returns, you should also

Reply:
@TDu ping

Reply:
@rousseldenis Thanks for the review, fixed.